packages/kit/src/exports/vite/dev/index.js

```diff
@@ -394,7 +394,8 @@ export async function dev(vite, vite_config, svelte_config) {
 			file === appTemplate ||
 			file === errorTemplate ||
 			file.startsWith(serviceWorker) ||
-			file.startsWith(hooks.server)
+			file.startsWith(hooks.server) ||
+			file.startsWith(path.resolve(cwd, 'src/tracing.server'))
```

Comment:
is this necessary? i don't think we need to run `sync.server` after adding/removing/changing this file

```suggestion
			file.startsWith(hooks.server)
```

Reply:
Just checked, it is not necessary